src/main/resources/templates/editMATLABBuildTask.ftl

```diff
@@ -2,3 +2,10 @@
 [@ww.select cssClass="builderSelectWidget" labelKey="executable.type" name="matlabExecutable"
 extraUtility=addExecutableLink  list=uiConfigSupport.getExecutableLabels('matlab') required='true'/]
 [@ww.textfield labelKey='matlab.build.tasks' name='buildTasks' cssClass="long-field" description="Specify the MATLAB build tasks to execute." required='false'/]
+[@ui.bambooSection]
+    [@ww.checkbox labelKey='matlab.options' name='optionsChecked'  toggle='true'/]
```

Comment:
I prefer this check box next to the 'Executable' text field. It contextually makes more sense there. User selects their desired MATLAB executable followed by their desired start-up options.

Reply:
Makes a lot of sense.

@sameagen-MW: We might want to revisit the Jenkin's UI based on where we land here.

Reply:
Also, what are the default options that get populated in this text field once the user selects this check box?

Reply:
Yeah, I think we'll have to be consisent across all of the UI-based platforms. And the default is the empty string, but it saves previously entered options in the case where a user unchecks and then rechecks that option.